### PR TITLE
#2846. Add unquoted imports tests for keywords

### PR DESCRIPTION
**Pull request by sgrekhov**

### CODE REVIEW

File: LanguageFeatures/Unquoted-imports/keywords_A01_t03.dart [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+///
+/// @description Checks that `OTHER_IDENTIFIER` can be used as a path
+/// components.
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=unquoted-imports
+
+import '../../Utils/expect.dart';
+import test_package2/keywords/async as l01;
+import test_package2/keywords/augment as l02;
```
Comment (eernstg) [inline LanguageFeatures/Unquoted-imports/keywords_A01_t03.dart:16]: This one is now built-in.

**Review by eernstg — COMMENTED** [review]
Looks good! Just one thing: `augment` is now a built-in identifier (so they say ;-).

**Comment by sgrekhov** [comment]
Thank you. Fixed. PTAL

**Review by eernstg — APPROVED** [review]
LGTM

**Pull request merged**